config/main.py

```diff
@@ -12,16 +12,50 @@
 from swsssdk import ConfigDBConnector
 from natsort import natsorted
 from minigraph import parse_device_desc_xml
+from portconfig import get_port_config
 
 import aaa
 import mlnx
 
-SONIC_CFGGEN_PATH = "sonic-cfggen"
+PLATFORM_ROOT_PATH = '/usr/share/sonic/device'
+SONIC_CFGGEN_PATH = '/usr/local/bin/sonic-cfggen'
+HWSKU_KEY = 'DEVICE_METADATA.localhost.hwsku'
+PLATFORM_KEY = 'DEVICE_METADATA.localhost.platform'
 
 #
 # Helper functions
 #
 
+# Returns platform and HW SKU
+def get_platform_and_hwsku():
```

Comment:
since this script talks directly with configuration database, it is better to retrieve the information e.g. HWSKU and platform from the database rather than calling commands

Reply:
👍 I can use PORT attribute in config_db to fetch both default and alias interfaces of each port. So, _get_hwsku() and _get_platform() are not necessary.